src/utils/slugify.js

```diff
@@ -0,0 +1,11 @@
+export default function slugify(word) {
```

Comment:
Damn, I thought we left this bad boy behind!

Reply:
behind where?

Reply:
@KhadijaMahanga Clemence and I had many disagreements of using slugify. But without slugify you would have to create slugs in all chart definitions.